From 90703859ec76bb2d13115e21596ec182ef0b2c52 Mon Sep 17 00:00:00 2001
From: Wang Hongcheng <Annie.Wang@amd.com>
Date: Tue, 22 Aug 2017 16:06:58 +0800
Subject: [PATCH 5427/5855] drm/amdkfd: fix bug device is not added to
 supported_devices

Add Tonga passthrough device 0x6929 to kfd device table.

BUG:SWDEV-130471

Signed-off-by: Wang Hongcheng <Annie.Wang@amd.com>
Reviewed-by: Xiangliang Yu <Xiangliang.Yu@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_device.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_device.c b/drivers/gpu/drm/amd/amdkfd/kfd_device.c
index 0da347a..e892998 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_device.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_device.c
@@ -277,6 +277,7 @@ static const struct kfd_deviceid supported_devices[] = {
 	{ 0x6920, &tonga_device_info   },	/* Tonga */
 	{ 0x6921, &tonga_device_info   },	/* Tonga */
 	{ 0x6928, &tonga_device_info   },	/* Tonga */
+	{ 0x6929, &tonga_device_info   },	/* Tonga */
 	{ 0x692B, &tonga_device_info   },	/* Tonga */
 	{ 0x692F, &tonga_vf_device_info   },	/* Tonga vf */
 	{ 0x6938, &tonga_device_info   },	/* Tonga */
-- 
2.7.4

